client/landing/jetpack-cloud/components/sidebar/index.jsx

```diff
@@ -0,0 +1,208 @@
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+/**
+ * Internal dependencies
+ */
+import Sidebar from 'layout/sidebar';
+import SidebarFooter from 'layout/sidebar/footer';
+import SidebarItem from 'layout/sidebar/item';
+import SidebarMenu from 'layout/sidebar/menu';
+import SidebarRegion from 'layout/sidebar/region';
+import ExpandableSidebarMenu from 'layout/sidebar/expandable';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+class JetpackCloudSidebar extends Component {
+	static propTypes = {
+		context: PropTypes.shape( {
+			path: PropTypes.string,
+		} ),
+	};
+
+	state = {
+		expandedSections: [],
+	};
+
+	componentDidMount() {
+		const section = this.getSectionFromCurrentPath();
+
+		if ( section ) {
+			this.toggleSection( section );
+		}
+	}
+
+	/**
+	 * Get current section name from path. When there's no section, we're in the dashboard.
+	 *
+	 * @returns {string} Current section
+	 */
+	getSectionFromCurrentPath() {
+		// If no section is present in the path, return 'dashboard'
+		return this.props.context.path.split( '/' )[ 2 ] ?? 'dashboard';
```

Comment:
Any chance that the path won't be specified?

Reply:
I assume that it's always in the context. However, adding adding a conditional chaining operator here isn't a bad idea I guess.

Reply:
This is outdated after pushing 603c49e.